src/components/SystemPreferences.vue

```diff
@@ -120,6 +120,9 @@ export default {
       this.$emit('warning', 'cpu', warningMessage);
       this.$emit('updateCPU', value);
     },
+    updatePort(value) {
```

Comment:
I don't think this is actually used?

Reply:
Yes, this was a holdover from when I tried to include port-updating in the SystemPrefs component before I realized Windows doesn't display it at all.